components/prow/manifests/nginx-controller.yaml

```diff
@@ -37,13 +37,6 @@ spec:
         ports:
         - name: backend-port
           containerPort: 8080
-        resources:
```

Comment:
any idea why was this here?

Reply:
this is from the official helm charts for `nginx-ingress` 
https://github.com/helm/charts/blob/a5f93f44c34fdb188919ac25b41423e9af718661/stable/nginx-ingress/values.yaml#L352-L358

but having those resource limitations was causing the pod to fail, I am not sure how the current `nginx-ingress` is running but It was failing for me every time the resource limitations were set.

I couldn't find why they were added there in the first place.

Reply:
ok , thanks for the extra details. I don't think we need these limitation as well